examples/jsm/lights/LightProbeGenerator.js

```diff
@@ -126,7 +127,18 @@ class LightProbeGenerator {
 
 	}
 
-	static fromCubeRenderTarget( renderer, cubeRenderTarget ) {
+	static async fromCubeRenderTarget( renderer, cubeRenderTarget ) {
+
+		let useAsync = false;
+		let flip = 1;
+
+		if ( renderer.isWebGPURenderer ) {
+
+			useAsync = true;
+
+			if ( renderer.coordinateSystem === WebGPUCoordinateSystem ) flip = -1;
+
```

Comment:
Nit. Perhaps the algorithm can be modified so `flip` can be 1 for WebGPU backend. That way we won't have to be flipping for eternity when WebGL is no longer supported. In other words, let WebGL be the odd case.

Reply:
will do tomorrow

Reply:
I think what @WestLangley suggested is to use `1` as the default value for `flip` and then check for WebGL and set the value to `-1`.